internal/cli/install.go

```diff
@@ -337,7 +338,17 @@ func (c *InstallCommand) Flags() *flag.Sets {
 			Hidden:  true,
 		})
 
-		for name, platform := range serverinstall.Platforms {
+		// Add platforms in alphabetical order. A consistent order is important for repeatable doc generation.
+		i := 0
+		sortedPlatformNames := make([]string, len(serverinstall.Platforms))
+		for name := range serverinstall.Platforms {
+			sortedPlatformNames[i] = name
+			i++
+		}
+		sort.Strings(sortedPlatformNames)
```

Comment:
We have this loop in three places now....is there a place where we can move it more directly into the docs generation so it always sorts alphabetically and these callers get the behavior automatically? Then future CLI packages get it "for free" too.

Reply:
I thought about that, but we usually want to preserve the order that plugins insert flags, so that they can set "global options" before "command options", so we can't alphabetize flag order for everyone.

This is also a weird case related only to server commands, where we have our own little internal plugin system for the different kinds of platforms you can install waypoint on. I seem to remember us thinking we weren't going to double down on this pattern in the future if possible.